cli/internal/services/authentication/authentication.go

```diff
@@ -108,7 +108,36 @@ func performTokenExchange(action string) error {
 // Starts the logout process.
 // This will delete the session and refresh token.
 func Logout() error {
-	return clearToken()
+	// Open the browser and redirect the user to the internal server
+	if err := utils.OpenBrowser(keycloakConfig.getLogoutURL(embeddedServerConfig.GetCallbackURL())); err != nil {
```

Comment:
Is the browser necessary here? There is no user input needed if I remember correctly

Reply:
If the user has to press a button opening the browser is perfectly valid and actually the preferred method imho.

So good work, very well thought out 👍